general.md

```diff
@@ -36,6 +37,11 @@ Der Spielleiter gibt kurze Details über die hingerichtete Person bekannt und de
 - Bürger - entscheiden das Spiel für sich, wenn alle Mafiosi tot sind.
 - Third-Party-Roles - gewinnen durch Erfüllung der jeweiligen Ziele in ihrer Beschreibung.
 
+
+Wir empfehlen, dass du dir das Video kurz anguckst, damit du eine ungefähre Vorstellung bekommst.
```

Comment:
Für eine grobe Einführung in das Spielprinzip und den Ablauf, empfehlen wir dieses Video: